pkg/shell/nav.jsx

```diff
@@ -88,7 +88,7 @@ export class CockpitNav extends React.Component {
                 let i = all.findIndex(item => item === cur);
                 i += step;
                 if (i < 0 || i >= all.length)
-                    document.querySelector("#" + sel + " .pf-c-text-input-group__text-input").focus();
+                    document.querySelector("#" + sel + " .pf-v5-c-text-input-group__text-input").focus();
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18806-20230516-073248-d32f52be-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)